src/DTO/NamedAccount.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Genkgo\Camt\DTO;
+
+use Genkgo\Camt\Iban;
+
+class NamedAccount extends Account
```

Comment:
We should not introduce a new class here. The spec defines `Acc` as type `CashAccount25`. And `CashAccount25` defines an object that has must have an ID (IBAN or Othr), and may have a name. Thus the name property must be added on the existing `IbanAccount`